hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -857,7 +857,7 @@
   </property>
   <property>
     <name>ozone.chunk.list.incremental</name>
-    <value>false</value>
+    <value>true</value>
```

Comment:
@jojochuang , I would suggest to change the "ozone.chunk.list.incremental" to "ozone.incremental.chunk.list",

to be consist with another property name "ozone.client.incremental.chunk.list".